src/Engine/ProtoCore/Lang/BuiltInMethods.cs

```diff
@@ -824,7 +824,8 @@ public BuiltInMethods(Core core)
                         new KeyValuePair<string, Type>("list", TypeSystem.BuildPrimitiveTypeObject(PrimitiveType.Var, Constants.kArbitraryRank))
                     },
                     ID = MethodID.GetKeys,
-                     MethodAttributes = new MethodAttributes(){Description = Resources.GetKeys}
+                     MethodAttributes = new MethodAttributes(true, false){Description = Resources.GetKeys}
```

Comment:
Perhaps this would make the usage clearer: `MethodAttributes = new MethodAttributes(hiddenInLibrary: true){Description = Resources.GetKeys}`

Reply:
👍